### PR TITLE
Pytorch backend

### PR DESCRIPTION
**Pull request by ninamiolane**

### CODE REVIEW

File: geomstats/hypersphere.py [hunk 1]
```diff
@@ -236,14 +236,6 @@ def log(self, point, base_point):
         mask_0_float = gs.cast(mask_0, gs.float32)
         mask_else_float = gs.cast(mask_else, gs.float32)
 
-        angle_0 = gs.boolean_mask(angle, mask_0)
```
Comment (johmathe) [inline geomstats/hypersphere.py]: why remove this?

File: tests/test_hypersphere.py [hunk 2]
```diff
@@ -124,6 +124,8 @@ def test_log_and_exp_general_case(self):
         expected = point
         expected = helper.to_vector(expected)
 
+        print(result)
```
Comment (johmathe) [inline tests/test_hypersphere.py]: leftover

File: tests/test_hypersphere.py [hunk 3]
```diff
@@ -315,6 +317,8 @@ def test_norm_and_dist(self):
         expected = self.metric.dist(point_a, point_b)
         expected = helper.to_scalar(expected)
 
+        #print('normresult', result)
```
Comment (johmathe) [inline tests/test_hypersphere.py]: leftover

File: geomstats/hypersphere.py (hunk 1 above)
Comment (ninamiolane) [reply geomstats/hypersphere.py]: it was not used. left over from a previous version probably

**Pull request merged**